src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-sun calendar--month-length-29">
+      <div class="day"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```
Fix everywhere

due to [BEM ](https://en.bem.info/methodology/quick-start/)